src/status_im/signals/core.cljs

```diff
@@ -53,8 +55,9 @@
                                              (and (some? account) (some? password)))
         multiaccount-data               (when received-account?
                                           (merge account {:password password}))
-        navigate-to-syncing-devices?    (and (or connection-success? error-on-pairing?) receiver?)
```

Comment:
@mohsen-ghafouri - the `error-on-pairing?` flag does include the connection error check too along with other errors.

Reply:
@smohamedjavid yes but if we had connection-error + other errors it will call navigate multiple times, so to avoid that i just want to check connection status